lib/commands/touch.js

```diff
@@ -0,0 +1,33 @@
+import log from '../logger';
+
+let commands = {}, extensions = {};
+
+commands.performActions = async function (actions) {
```

Comment:
there is no need for preprocessing. `return await this.winAppDriver.sendCommand('/actions', 'POST', {actions});` would be enough